packages/core/src/configuration.ts

```diff
@@ -50,6 +50,11 @@ export const legend: LegendOptions = {
 		radius: 6.5,
 		spaceAfter: 4,
 	},
+	truncation: {
+		type: TruncationTypes.END_LINE,
+		threshold: 16,
+		numCharacter: 14,
+	},
```

Comment:
maybe we can store this in a variable? standardTruncationOptions?

Reply:
Resolved. Thanks.